perun-cli/createService

```diff
@@ -15,13 +15,14 @@ sub help {
 	--description  | -d service description
 	--script       | -s service script path (default './service_name')
 	--enabled      | -e enabled (default 'true')
+	--expired      | -x use expired members (default 'false')
```

Comment:
We recently changed the `useExpiredMembers` parameter to have `true` as a default value in `perun-core` because most of the services use expired members. So I think it would be better to have it as `true` here as well.